pyartcd/pyartcd/pipelines/build_microshift.py

```diff
@@ -138,41 +138,61 @@ async def run(self):
                             self._logger.error(slack_err)
                     raise build_err
 
-                # Create a PR to pin microshift build
+                # Nothing to do for stream builds after this
+                if assembly_type is AssemblyTypes.STREAM:
+                    return
+
+                message = f"microshift for assembly {self.assembly} has been successfully built."
+                await self.slack_say(message)
+
+            # Check if we need create a PR to pin the build
+            pinned_microshift_rpms = util.get_rpm_if_pinned(releases_config, self.assembly, 'microshift')
```

Comment:
we should do the check earlier, if build exist don't need to rebase and build again

Reply:
Good point, I'll update

Reply:
Updated